asv_bench/benchmarks/timeseries.py

```diff
@@ -107,6 +107,8 @@ def time_infer_freq_daily(self):
     def time_infer_freq_business(self):
         infer_freq(self.b_freq)
 
+    def time_to_date(self):
+    	self.rng.date
```

Comment:
might as well as ones for ``self.rng.time()`` and ``self.rng.to_pydatetime()`` while at it here.

note that this should be ``self.rng.date()``

Reply:
DatetimeIndex.date and DatetimeIndex.time are tagged with the `@property` tag at the moment. Do you wante me to remove that tag so that they can be used as functions instead?

https://github.com/tmnhat2001/pandas/blob/e1b6d6230f54f3eee90225b25f8907989e3828df/pandas/core/indexes/datetimes.py#L1631

https://github.com/tmnhat2001/pandas/blob/e1b6d6230f54f3eee90225b25f8907989e3828df/pandas/core/indexes/datetimes.py#L1640

Reply:
sorry that was a typo;